runtime/rasdump/javadump.cpp

```diff
@@ -2051,6 +2056,20 @@ JavaCoreDumpWriter::writeHookInterface(struct J9HookInterface **hookInterface)
 			}
 			_OutputStream.writeCharacters("NULL\n");
 		}
+		//reset the eventDump statistics
+		eventDump->count = 0;
```

Comment:
This represents a change in behavior that not all users will want. What's the motivation for resetting those statistics?

Reply:
Motivation.... Relevant (typically, relatively recent) stats may get 'polluted' with irrelevant old data from long runs. Especially true for longestHook stats and to some extent for totalTime (and average which can be deduced). 

Backward compatibilitiy..... Yes, it's change of behavior. But, if we reset, and we have all reports/dumps available, we can reconstruct those numbers for the whole run, if really needed.

Affected users.... This is relatively new feature and rarely used, and I believe not documented. Typically, by us, designers, and Service team. We consulted @manqingl from Service and he was ok with it.

Reply:
Yes, I requested this change from service perspective.